src/Contracts/NovaSelectEnumInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Idez\Caradhras\Contracts;
+
+interface NovaSelectEnumInterface
+{
+    public function label(): string;
+
+    public static function novaOptions(): array;
```

Comment:
Como trata-se de um pacote, acredito que ele deva ser mais abstrato. Mas estou aberto a discussão

```suggestion
interface SelectEnumInterface
{
    public function label(): string;

    public static function options(): array;
```